matRadGUI.m

```diff
@@ -2451,7 +2451,8 @@ function getPlnFromGUI(handles)
 
 try
     cst = evalin('base','cst');
-    if sum(strcmp('TARGET',cst(:,3))) > 0 && get(handles.checkIsoCenter,'Value')
+    if (sum(strcmp('TARGET',cst(:,3))) > 0 && get(handles.checkIsoCenter,'Value')) || ...
+            (sum(strcmp('TARGET',cst(:,3))) > 0 && ~isfield(pln,'isoCenter'))
        pln.isoCenter = ones(pln.numOfBeams,1) * matRad_getIsoCenter(cst,ct);
```

Comment:
Yes that's correct, I thought it might be more consistent if we one day change behaviour to use pln as input argument for getPlnFromGUI as well. Unfortunately I encounter problems when I 
- change couch angle in GUI

because then getPlnFromGUI is called and as you said, the statement is always true, as opposed to when I 

- change isoCenter, a seperate function is called which does not call getPlnFromGUI

is there a reason for not just calling getPlnFromGUI for all edits in GUI (regarding plan) and using pln variable if it exists?

But I am kind of  aware, I actually wanted to fix the issue of unwanted overwrite of the iso center  but now there is a new one ...